### PR TITLE
Eb 1478 create driver async

### PR DESCRIPTION
**Pull request by vinay3206**

### CODE REVIEW

**Comment by vinay3206** [comment]
@maartenhemmes  reset password related tests are failing here

File: src/data/customer/__test__/driver.test.js [hunk 1]
```diff
@@ -181,65 +195,70 @@ describe('Create new driver ', async () => {
...
-      vehicleBrand: 'Scooter',
-      vehicleModel: '12456',
-      vehicleTypeId: 1,
-    };
+    it('Test for creating the driver schedule', async () => {
+      const payload = {
+        driverId: driver.id,
+        transactionGroupId: 180,
+        windows: [
+          {
```
Comment (maartenhemmes) [inline src/data/customer/__test__/driver.test.js]: @vinay3206 should we be testing including the recursion ID and also check if the 'default value' is there?

File: src/data/customer/__test__/driver.test.js [hunk 2]
```diff
@@ -181,65 +195,70 @@ describe('Create new driver ', async () => {
...
-      expect('id' in responseCreatedriver).toBeTruthy();
+      schedule = await createDriverScheduleAsync(payload, token.accessToken);
+
+      expect('data' in schedule).toBeTruthy();
+    });
 
+    it('Test for updating the driver schedule', async () => {
       const payload = {
-        driverId: responseCreatedriver.id,
+        driverId: driver.id,
```
Comment (maartenhemmes) [inline src/data/customer/__test__/driver.test.js]: where does this driver.id come from? before I used the response object from the 'create' @vinay3206

Comment (vinay3206) [reply src/data/customer/__test__/driver.test.js]: @maartenhemmes  it is set from the first test case on this file for creating driver.

File: src/data/customer/__test__/driver.test.js (hunk 1 above)
Comment (vinay3206) [reply src/data/customer/__test__/driver.test.js]: @maartenhemmes there is no recursion or default while creating the driver schedule.
I mean frontend won't be using it with recursion.

**Pull request merged**